### PR TITLE
Add release notes for edge-19.7.5

### PR DESCRIPTION
**Pull request by ihcsim**
Signed-off-by: Ivan Sim <ivan@buoyant.io>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
+## edge-19.7.5
+
+* CLI
+  * Improved the installation report produced by the `linkerd check` command
+    to include the control plane pods' live status
```
Comment (wmorgan) [inline CHANGES.md]: Maybe "liveness statuses" here, but I'll defer to resident Linkerd Grammatician @scottcarol

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+  * Added the `--all-namespaces` (`-A`) option to the `linkerd get`,
```
Comment (siggy) [inline CHANGES.md]: The change was just adding the `-A` shortcode, this sounds more like we added `--all-namespaces` support in general, maybe something like:
```
* Added `-A` shortcode for the `--all-namespaces` option in `linkerd get`, `linkerd edges`, and `linkerd stat`.
```

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+  * Fixed an issue with discovering StatefulSet pods via their unique hostname
+  * Fixed an issue with traffic split where outbound proxy stats are missing
+  * Bumped Prometheus to 2.11.1
+  * Bumped Grafana to 6.2.5
+  * Upgraded the service profile CRD to v1alpha2 where the openAPIV3Schema
+    validation is replaced by a validating admission webhook. No changes
+    required for users currently using v1alpha1
+  * Updated the control plane's pod security policy to restrict workloads from
+    running as `root` in the CNI mode (thanks @codeman9!)
+  * Introduced cluster heartbeat cron job
```
Comment (siggy) [inline CHANGES.md]: this isn't very user-facing, though i do appreciate being transparent about it. thoughts @wmorgan ?

Comment (wmorgan) [reply CHANGES.md]: Mild preference that we keep it in

File: CHANGES.md (hunk 1 above)
Comment (scottcarol) [reply CHANGES.md]: "Live status" works for me, seems simpler

**Pull request merged**